NEWS

```diff
@@ -1,3 +1,18 @@
+08-Jan-2021 Ignition 2.9.0
+
+  Changes:
+
+    - Require filesystems.format if wipeFilesystem or mountOptions is specified
+    - Refactor code to address golangci-lint warnings
+
+  Bug Fixes:
+
+    - Fix fetching configs from S3 resources when running on non-default AWS partitions
+    - Support fetching userdata from IMDSv2 on AWS
+    - Fix crash on partitions with no number or label
+    - Correctly document storage.filesystems.path as optional
+    - Clarify semantics of systemd.units.name
```

Comment:
Since this is just a docs change, `Clarify documented semantics` would be clearer.